js/index.js

```diff
@@ -1,7 +1,71 @@
 // Iteration 1: Names and Input
+console.log("I'm ready!");
 
+let hacker1 = "Victoria";
+
+console.log(`"The driver's name is ${hacker1}."`);
```

Comment:
In this case and in the hacker2 version the double quotes are unnecessary :) the backticks are already giving you a string. :)